File: Java/medidas.java
import java.util.Locale;
import java.util.Scanner;

public class medidas {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        Locale.setDefault(Locale.US);

        System.out.print("Digite a medida A: ");
        double a = sc.nextDouble();
        System.out.print("Digite a medida B: ");
        double b = sc.nextDouble();
        System.out.print("Digite a medida C: ");
        double c = sc.nextDouble();

        double quadrado = a * a;
        double triangulo = a * b / 2;
        double trapezio = (a + b) * c / 2;

        System.out.printf("Area do quadrado = %.4f %n", quadrado);
        System.out.printf("Area do triângulo = %.4f %n", triangulo);
        System.out.printf("Area do trapézio = %.4f %n", trapezio);

        sc.close();

    }
}

File: C/medidas.c
#include <stdio.h>

double a, b, c, quadrado, triangulo, trapezio;

int main() {
    printf("Digite a medida A: ");
    scanf("%lf", &a);
    printf("Digite a medida B: ");
    scanf("%lf", &b);
    printf("Digite a medida C: ");
    scanf("%lf", &c);

    quadrado = a * a;
    triangulo = a * b / 2;
    trapezio = (a + b) * c / 2;

    printf("AREA DO QUADRADO = %.4lf\n", quadrado);
    printf("AREA DO TRIANGULO = %.4lf\n", triangulo);
    printf("AREA DO TRAPEZIO = %.4lf\n", trapezio);
}
